core/storage/question/gae_models.py

```diff
@@ -670,6 +677,9 @@ class QuestionSummaryModel(base_models.BaseModel):
         indexed=True, required=True)
     # The html content for the question.
     question_content = ndb.TextProperty(indexed=False, required=True)
+    # The misconceptions attached to the question.
+    misconception_ids = ndb.JsonProperty(
+        default=[], indexed=True, required=True)
```

Comment:
repeated=True, indexed=True (and use StringProperty)

Reply:
FYI, I tried using string property with repeated=True but turns out that didn't allow empty arrays. It is possible to have empty arrays for this field if the skill has no misconceptions.

Reply:
I'm pretty sure that's not correct. Did you use required=True or something like that as well?

Reply:
You are right, it works. Done.